script.sh

```diff
@@ -177,12 +178,28 @@ fi
 
 cat >>Dockerfile <<EOF
 RUN apt-get update && apt-get dist-upgrade --yes
-RUN apt-get install --yes --no-install-recommends build-essential equivs devscripts git-buildpackage
+RUN apt-get install --yes --no-install-recommends build-essential equivs devscripts git-buildpackage apt-transport-https wget ca-certificates
 
 WORKDIR $(pwd)
 COPY . .
 EOF
 
+if [ -n "${TRAVIS_EXTRA_REPOSITORY_GPG_URL}" ]
+then
+	cat >>Dockerfile <<EOF
+RUN wget -O- "${TRAVIS_EXTRA_REPOSITORY_GPG_URL}" | apt-key add -
+EOF
+fi
+
+if [ -n "${TRAVIS_EXTRA_REPOSITORY}" ]
+then
+	cat >>Dockerfile <<EOF
+RUN echo "deb ${TRAVIS_EXTRA_REPOSITORY}" >> /etc/apt/sources.list
+RUN echo "deb-src ${TRAVIS_EXTRA_REPOSITORY}" >> /etc/apt/sources.list
+RUN apt-get update
```

Comment:
> Please do

Sorry, what do you want me to do? The changes I mentioned above I already implemented a while ago.

I only noticed that the PR needed a rebase, so I've made it up-to-date.